mapbox/services/geocoding.py

```diff
@@ -75,7 +81,7 @@ def reverse(self, lon=None, lat=None, types=None):
         `response.geojson()` returns the geocoding result as GeoJSON.
         `response.status_code` returns the HTTP API status code.
 
-        See: https://www.mapbox.com/developers/api/geocoding/#reverse."""
+        See: https://www.mapbox.com/api-documentation/#geocoding."""
```

Comment:
More specific URL might be helpful: https://www.mapbox.com/api-documentation/#retrieve-places-near-a-location